### DISCUSSION TITLE
Voice and tone

### DISCUSSION THREAD

**Discussion by Sarah-Lombardi** [UKHomeOffice/design-system#600] [Ideas]
Do we need to develop a Home Office tone and voice?

**Comment by Sarah-Lombardi** [comment]
Here is a podcast and blog around the work Co Op Funeral Care did on tone and voice. [The principles that guide our content design and communications in Funeralcare](https://digitalblog.coop.co.uk/2021/04/15/the-principles-that-guide-our-content-design-and-communications-in-funeralcare/)

[Ep. 36: Helen Lawson, Co-op Funeral - The Content Strategy Podcast](https://www.contentstrategy.com/episodes/36-helen-lawson-co-op-funeralcare-the-importance-of-empathetic-language-voice-and-tone)

Also the only reference I can currently find on [GOV.UK](http://gov.uk/) to Tone of voice is in the background research for their content principles (2013).

‌

Plain English and ‘tone of voice’
Tone of voice
‘Tone of voice’ is particular to a company, service or other organisation. It represents their values and works with elements of visual presentation, such as typefaces and use of typographic styles, as well as vocabulary, mode of address, and other linguistic considerations.

Delin (2005) describes tone of voice as helpful in engaging people with content ‘perhaps about products and services they wish to buy but also about benefits and services that they are invited to take part in or claims, such as pensions, tax credits, or advice, health services and screenings, safety information and more’.

Much research on writing for the web supports an informal, conversational style of writing, including contractions and personal pronouns, that has evolved from the tradition of computer dialog between computer and reader (Carliner, 1990; Evans and McBride, 2004).

Despite recommendations for relative informality, Delin’s reference to work with an unnamed (for confidentiality reasons) government department indicates some of the challenges in finding the right tone of voice. Participants in her small survey thought that information presented in the simplest terms was too friendly for a government agency – and therefore came across as false.

They did not want to see contractions such as you’re and we’ll; they preferred ‘please call us’ to ‘give us a ring’; they liked ‘you are likely to be entitled to a further amount’ but thought ‘that you can get more money’ was offensive because it suggested they were childish and greedy.

The study participants preferred language that was direct but not too chatty. They preferred, for example: complete to fill in; receive to get. Delin’s work suggests that readers may be particularly sensitive to words that are used in relation to benefits or transactions, with attention to the challenges of writing in a way that is intended to be direct and conversational, yet might be perceived as too colloquial and therefore inappropriate.

One study has suggested, for example, that some ethnic groups may be offended by use of direct ‘you’ (Rose, 1981), and recommends not repeating ‘you’ if usage seems to be getting very insistent. Black and Stanbridge (2012) have found users can also react negatively to a writing style that over-uses motivational features and can interpret this as patronising or inappropriate style. A balance is needed that takes into account the different levels of understanding and expectations readers bring to a text.

**Comment by Sarah-Lombardi** [comment]
From Steve Shukor: 
Some thoughts and suggestions:

Should we make the distinction between voice and tone? You have the same voice all the time, but your tone may change depending on the situation.

Also, I feel voice and tone should not be confused with writing plain language principles.

Voice suggestions:

respectful: we are polite, inoffensive and sensitive to our users needs, backgrounds and circumstances. This shows that we care and want to help.

plainspoken (or factual/straightforward): our users have accessibility needs or may not speak English as a first language. We want to be inclusive. We use plain English and avoid colourful language, technical terms, humour and colloquialisms.

compassionate: we need to be sensitive to the experiences and needs of all our users. (sympathy vs empathy?)

inclusive: we should not write in a way that could make some think or feel 'this is not for me'.

Tone suggestions:

conversational but not chatty, casuakl or informal: this makes sounds more like humans but we need to speak with authority. (language flows better, more human, avoid contractions and expressions that may not translate well or that could be misinterpreted. Personal rather than formal - use 'you', demonstrates we are trying to help people with their specific needs, treat people as individuals.) No slang, emojis, or exclamation marks or contractions.

neutral: we provide the facts without giving an opinion or judgement.

(An organisation should have one voice but the tone should vary depending on the context.)

Sources:

[What is tone of voice in communication? (With definition)](https://uk.indeed.com/career-advice/career-development/tone-of-voice-in-communication)

[Voice and Tone | Mailchimp Content Style Guide](https://styleguide.mailchimp.com/voice-and-tone/)

[The Four Dimensions of Tone of Voice](https://www.nngroup.com/articles/tone-of-voice-dimensions/)

[Casual or conversational? A guide to writing formality](https://blog.outwrite.com/a-guide-to-writing-formality/#:~:text=There%20are%20many%20names%20for,between%20formal%20and%20informal%20writing.)

**Comment by Sarah-Lombardi** [comment]
Agreed to making a distinction between voice and tone:

* Voice is based on our Home Office values, for example honesty, respect and compassion
* Tone is dependent upon user need and channel and I would like to get some examples to demonstrate this

I’m currently contacting various people to get examples of where tone has needed to change. I am speaking to someone in comms about how a letter might change in response to inclusion research on evisas, and also will speak to Metis content designers on how tone has been adjusted to be more conversational on their Ask Metis assistant.

**Comment by Sarah-Lombardi** [comment]
Drafted and shared with comms early August. Ready for sign off by Home Office Style Council.

**Reply by charleemg** [reply]
This is looking good! I'm glad you are thinking about clear writing and plain English principles. I would possibly include a more detailed explanation of active versus passive voice, as I think it's really crucial in clear communication in government. It actually ties up very well with the "empowering" pillar, as sentences written in active voice tend to focus naturally on the positives, and what people can do!

I'm finding the voice conversation very interesting - as an ex-GDSer, it's not really something we talked about much there, as we standardised our voice across everything, internally and externally, for consistency. I am trying to work out if tone never really came up because the tone sort of wrote itself, or maybe we missed a trick?

I wonder about the phrase " 'You can work and study, and come and go as many times as you want" as a specific example, though, as I wonder if "come and go as many times as you want" may be a little colloquial for somebody who doesn't have English as a first language. While I can understand it's aimed at being personal and empathetic, I think it's important to also keep all government language simple and direct (average reading age of a UK citizen is between 9 and 11!).

I've recently delivered some training on clear communication for Home Office enterprise architects, so please let me know if I can help in any way. Thanks!

**Reply by Sarah-Lombardi** [reply]
Thanks so much for this feedback! I’ll look into how I can make the active voice clearer as you’re right it contributes to many of the elements of our voice. 

I was surprised there wasn’t a Gov.uk voice and yet it had infused into what we did. No one could believe it didn’t exist! It shows how successful the other guidance is. 

And I’ll see if anyone has other examples we can share. Or I could remove the “come and go” part which you’re right could be a little colloquial. 

Are you still working with us at Home Office?

**Reply by charleemg** [reply]
Hiya, yes I'm currently working with the Enterprise Architecture team on the IT strategy blueprints. I'll pop you a Slack message and say hello.
